README.md

```diff
@@ -13,6 +13,16 @@ This repository contains the [source code](./src) and the [content](./content) f
 
 Read through the [contributing guide](CONTRIBUTING.md) to learn how you can contribute to the Prisma documentation.
 
+## Prerequisites for macOS Monterey (Apple M1 chip)
```

Comment:
Is this a prereq for Monterey or running on M1?

Reply:
The M1 chip itself... yeah so we could just not mention the OS version... though it is helpful to remind people of which OS that chi[p started appearing in.

@zachtil the heading could be something like ## Prerequisites for Apple M1 chip (macOS Monterey and later)

Reply:
M1s first started on Big Sur (Nov 2020), and Monterey was released in 2021.

I've just tested on my old Macbook (2013 model) which runs the now 2-year old Catalina and `npm install` just worked, so the Intel machines seem clear.

Is it worth it test on an M1 machine that still runs Big Sur just to exclude the OS variable?

Reply:
Seems to be definitely related to M1 and the fact that sharp requires vips - https://stackoverflow.com/questions/54409953/cant-install-sharp/56108335.

I'll modify as suggested, just indicate "macOS BigSure and later".